diagnostic.md

```diff
@@ -57,13 +67,15 @@ Please write a method that traverses from a list item (`li`) DOM node to the
 body DOM node, then hides it.
 
 <!-- your answer starts here -->
-
+$('li').hide('li')
 <!-- your answer ends here -->
 
 ## Question 8
 
 Use jQuery to register a click handler for clicks inside `div` items.
 
 <!-- your answer starts here -->
+$('div').on('click', function (event) {
```

Comment:
your function doesn't need event as an argument i.e. it can be an anonymous function : 

```
 $('div').on('click', () => {

  });
```